android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMPBrowserActivity.java

```diff
@@ -40,6 +40,16 @@
   // URL for keyboard search web page presented to user when they add a keyboard in the app.
   private static final String KMP_SEARCH_KEYBOARDS_FORMATSTR = "https://%s/go/android/%s/download-keyboards%s";
   private static final String KMP_SEARCH_KEYBOARDS_LANGUAGES = "/languages/%s";
+
+  // Patterns for determining if a link should be opened in external browser
+  // 1. Host isn't keyman.com (production/staging)
+  // 2. Host is keyman.com but not /keyboards/
+  private static final String INTERNAL_KEYBOARDS_LINK_FORMATSTR = "^http(s)?://(%s|%s)/keyboards(\\/)?(.+)?$";
```

Comment:
I see you are escaping one of the forward slashes but not the other two? Do we need to escape them?

Reply:
good point. yep, I can remove the escaping slash

Reply:
... so it turns out I need something like `(/?.*)?$`.  Testing and another commit to follow